### ISSUE TITLE
Label appears below Select on version upgrade from 5.31.0 to 6.0.0

### ISSUE THREAD

**Issue by cknabe** [carbon-design-system/carbon-components-react#993]
## Detailed description

> Describe in detail the issue you're having.
On upgrading to version 6.4.2 from 5.31.0 one of the Select labels is showing below the Select now.

> Is this a feature request (new component, new icon), a bug, or a general issue?
Bug

> Is this issue related to a specific component?
Select

> What did you expect to happen? What happened instead? What would you like to see changed?
I would like to be able to fix it so that the Label shows up above the component.

> What browser are you working in?
Chrome, Firefox

> What version of the Carbon Design System are you using?
I upgraded to:
    "carbon-addons-cloud": "^2.0.1",
    "carbon-addons-cloud-react": "^1.1.4",
    "carbon-components": "^8.16.8",
    "carbon-components-react": "^6.4.2",
    "carbon-icons": "^7.0.6",

from:
"carbon-components": "8.16.8",
"carbon-components-react": "5.31.0",
"carbon-icons": "^6.1.0",

> What offering/product do you work on? Any pressing ship or release dates we should be aware of?
Gen1 order form.
The Select in here is the problem: https://github.ibm.com/Bluemix/Gen1.VSI/blob/master/src/client/containers/Network/PortSpeed.js

## Steps to reproduce the issue
I cannot reproduce it outside of this specific location ^

Uplink Port Speed shows below the Select it labels
![image](https://user-images.githubusercontent.com/13582620/41299198-92675ee0-6e28-11e8-9f2f-a178736ab3a4.png)

**Comment by cknabe** [comment]
I tested it at 6.0.0 and it shows up below for that version as well.

**Comment by marijohannessen** [comment]
Is there a live version of this anywhere? It's probably something in the CSS that's being overwritten, so we'd need to inspect that to help out with this further.

**Comment by cknabe** [comment]
@marijohannessen Thanks for the reply! I do not have a live version available since we do development locally. I think I could possibly push it up to Bluemix in a private org, but that would be a lot of work. 

For now, I will continue to look into it. I did a quick glance at our component and I did not see any additional CSS added on the component itself. Looking at the CSS in the browser, nothing jumps out that would explain why the label is actually listed below the select in the HTML.

![image](https://user-images.githubusercontent.com/13582620/41302223-9067c966-6e2f-11e8-880f-bb58d98116cd.png)
